lib/tbot/config/destination_directory.go

```diff
@@ -126,6 +128,9 @@ func mkdir(p string) error {
 
 		log.Infof("Created directory %q", p)
 	} else if err != nil {
+		if errors.Is(err, fs.ErrPermission) {
+			log.Errorf("Teleport does not have permission to write to: %v. Ensure that you are running as a user with appropriate permissions.", p)
```

Comment:
This might confuse a user, as you'll log the error here, but also return a [different] error, which will eventually also be logged.

In general you should handle an error either by logging it or returning it, but not both.